src-bindings/fs-extra/fsextra_stub.js

```diff
@@ -0,0 +1 @@
+joo_global_object.fsExtra = require('fs-extra')
```

Comment:
It's a shame we have to add a dependency and bindings to the library for a single function. Would it be easy to implement copySync with of standard NodeJS functions?

Reply:
Thanks for the quick review :))

https://github.com/jprichardson/node-fs-extra/blob/master/lib/copy/copy-sync.js 🤷 

To reduce the number of files perhaps bundling it into the existing `FS` bindings might be a good idea seeing as they're already an altered form of the "true" Node FS module ?

Reply:
> To reduce the number of files perhaps bundling it into the existing FS bindings might be a good idea seeing as they're already an altered form of the "true" Node FS module ?

Sure, that sounds good 🙂

Reply:
Actually, unless I'm mistaken, wouldn't this require an extra dependency for the package as `fs-extra` is only a `dev` dependency at the moment https://github.com/ocamllabs/vscode-ocaml-platform/blob/master/package.json#L966 ? Maybe not worth it then?

Reply:
Do we even need this dependency?

I noticed you added `Fs.Sync.copy`, can't that be used instead of `Fs_extra.copy_sync`?

Reply:
IIUC `Fs_extra.copy_sync` is used because it copied directories (not just that it is synchronous).

Reply:
Okay in that case I personally think we should leave it as-is.

> It's a shame we have to add a dependency and bindings to the library for a single function.

@tmattio: It's not _adding_ a dependency since the JS version of the tests also used this library. 
We can remove the dependency later in another PR if necessary.

> unless I'm mistaken, wouldn't this require an extra dependency for the package as fs-extra is only a dev dependency at the moment

That's right. That means that keeping the bindings to `fs-extra` separate would not require us to add it as a runtime dependency.